saleor/graphql/core/types/common.py

```diff
@@ -711,6 +711,10 @@ class TranslationError(Error):
     code = TranslationErrorCode(description="The error code.", required=True)
 
 
+class TranslationBulkError(BulkError):
+    code = TranslationErrorCode(description="The error code.", required=True)
```

Comment:
I believe that for new mutations we should always have separate Error and separate error code (Even if the new mutation uses the same as the generic one).
Providing a generic one cause the frontend developer doesn't know which exact error can be raised by specific mutation, so she/he needs to provide the implementation for all of them.
Also if the bulk mutation that you introduce will need a new error code, we will have to add it to the generic `TranslationErrorCode`, which means that from a schema perspective it will look like all existing mutations can raise the error code that you introduced only to a bulk mutation.